kml/kml_tests/gpx_tests.cpp

```diff
@@ -125,8 +125,8 @@ UNIT_TEST(Route)
   auto line = dataFromFile.m_tracksData[0].m_geometry.m_lines[0];
   TEST_EQUAL(line.size(), 2, ());
   TEST_EQUAL(dataFromFile.m_categoryData.m_name[kDefaultCode], "Some random route", ());
-  TEST_EQUAL(line[0], mercator::FromLatLon(48.20984622935899, 16.376023292541507), ());
-  TEST_EQUAL(line[1], mercator::FromLatLon(48.209503040543545, 16.381065845489506), ());
+  TEST_EQUAL(line[0], geometry::PointWithAltitude(mercator::FromLatLon(48.20984622935899, 16.376023292541507), 184), ());
```

Comment:
Are fractional values of altitude supported? Let's test it too (maybe edit the source/test GPX file?)

Reply:
It is possible to have it fractional in GPX file, but for now altitude is integer (geometry/point_with_altitude.hpp:9) and for kml we round it to int: kml/serdes.cpp:644.

Reply:
@vng KML, GPX, and real altitude always have/support fractional values. It may be important in some cases. Why elevation was stored as int16_t? Can we change it at least to float?

Reply:
Compression is better and simpler. You will be surprised how big an average altitude error of regular GPS receivers is :) So these 0.5 meters are about nothing, when we talk about 10nth of meters errors.

Reply:
@vng 
1. What kind of compression are you talking about? The one that is used in our mwm files? It should not be related in any way to externally recorded elevation data from GPS.
2. I know that most smartphones provide several meters of vertical accuracy on average in good conditions. Although there are always exceptions, and there are augmented GPS receivers with a sub-meter accuracy.
3. Different users will use OM in different scenarios, with different quality satellite receivers. It is always better to store the original raw data for future exporting and processing instead of rounding it to the nearest integer.
4. I don't see any issues if we use another Point class with a more precise altitude encoded as a float. The only obvious overhead is 2 bytes more memory space and 6+ bytes more disk space (KML) for each measurement (that can be optimized for no-elevation cases). For 1 hour of recording, it is an additional 7kb or 21kb of memory.

We can finish this PR without changing the existing point class and create a separate issue for it to implement later. IMO it is important for our future track recorder implementation.